app/script/view_model/bindings/CommonBindings.js

```diff
@@ -174,7 +174,7 @@ ko.bindingHandlers.heightSync = {
 
     const resizeTarget = () => {
       const sourceHeight = element.scrollHeight;
-      const targetHeight = parseInt(targetElement.style.height, 10);
+      const targetHeight = targetElement.offsetHeight;
```

Comment:
Ok, here is the **actual** fix.

The problem here is that we would trigger resizing on the first render of the input bar (because no CSS property has been set yet) and thus trigger a forced scrollToBottom on the message list.

Using the actual height (not the CSS property) avoids triggering the callback and the scrolling. 

On a side note: This addresses this specific bug but is actually the consequence of multiple view models accessing the same DOM element directly, we **should never do that** in the future.